eahub/templates/account/login.html

```diff
@@ -1,70 +1,84 @@
 {% extends 'base_bs5.html' %}
 
-{% load crispy_forms_field crispy_forms_tags %}
+{% load crispy_forms_field crispy_forms_tags socialaccount static %}
 
 {% block navbar %}
-    {% include 'components/navbar.html' %}
+  {% include 'components/navbar.html' %}
 {% endblock navbar %}
 
 {% block title %}Login{% endblock %}
 
 {% block content %}
-    <div class="container account-floating-form">
-        <h1>Login</h1>
-
-        {% include 'includes/messages.html' %}
-    
-        <form method="POST" action="{% url 'account_login' %}">
-            {% csrf_token %}
-
-            {% if form.errors %}
-                <div class="form-group mb-4">
-                    {% for error in form.non_field_errors %}
-                        <p class="error">{{ error|escape }}</p>
-                    {% endfor %}
-                    {% for field in form %}
-                        {% for error in field.errors %}
-                            <p class="error">{{ error|escape }}</p>
-                        {% endfor %}
-                    {% endfor %}
-                </div>
-            {% endif %}
-
-            <div class="form-floating">
-                <input type="email"
-                    class="form-control form-control-lg"
-                    name="{{ form.login.name }}"
-                    placeholder="Name"
-                    id="{{ form.login.id }}"
-                    required>
-                <label for="{{ form.login.id }}">Email</label>
-            </div>
-
-            <div class="form-floating">
-                <input type="password"
-                    class="form-control form-control-lg"
-                    name="{{ form.password.name }}"
-                    placeholder="Email"
-                    id="{{ form.password.id }}"
-                    required>
-                <label for="{{ form.password.id }}">Password</label>
-            </div>
-
-            {% if form.captcha %}{% crispy_field form.captcha %}{% endif %}
-
-            <div class="fs-5 additional-links">
-                <a href="{% url 'account_reset_password' %}">
-                    Forgot password?
-                </a>
-            </div>
-
-            <input class="btn btn-primary btn-lg" type="submit" value="Login">
-        </form>
+  <div class="container account-floating-form">
+    <h1>Login</h1>
 
+    {% include 'includes/messages.html' %}
+
+    <div class="join__sso join__sso-with-margin">
+      <a href="{% provider_login_url 'google' %}">
+        <div class="sso__google-btn">
+          <img class="sso__icon"
+            src="{% static 'global/images/home/google.svg' %}"
+          >
+          <span>Sign In with Google</span>
```

Comment:
Minor: Not sure what's common, but is it right to use the same phrase "Sign in" both for signing up and logging in?